dht_net.go

```diff
@@ -178,6 +179,7 @@ func (ms *messageSender) SendMessage(ctx context.Context, pmes *pb.Message) erro
 }
 
 func (ms *messageSender) writeMessage(pmes *pb.Message) error {
+	ms.s.SetWriteDeadline(time.Now().Add(dhtMessageTimeout))
```

Comment:
All writes are pretended with it and clearing it would cause race where one write ends and another starts and the starting one gets it deadline cleared.